tools/checklocks/state.go

```diff
@@ -166,20 +228,23 @@ func (l *lockState) store(addr ssa.Value, v ssa.Value) {
 // isSubset indicates other holds all the locks held by l.
 func (l *lockState) isSubset(other *lockState) bool {
```

Comment:
It is not clear if this name is still appropriate. This is now a bit misleading. Zvonimir, suggestions?

Reply:
Yeah, just subset is not technically correct anymore I believe. Maybe something like "holdsWhenever" or even "heldSubset"?